content/en/blog/_posts/2022-12-15-dynamic-resource-allocation-alpha/index.md

```diff
@@ -19,11 +19,11 @@ Third-party resource drivers are responsible for interpreting these parameters
 as well as tracking and allocating resources as requests come in.
 
 Dynamic resource allocation is an *alpha feature* and only enabled when the
-`DynamicResourceAllocation` [feature
-gate](/docs/reference/command-line-tools-reference/feature-gates/) and the
-`resource.k8s.io/v1alpha1` {{< glossary_tooltip text="API group"
-term_id="api-group" >}} are enabled. For details, see the
-`--feature-gates` and `--runtime-config` [kube-apiserver
+`DynamicResourceAllocation`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and the
+`resource.k8s.io/v1alpha1`
+{{< glossary_tooltip text="API group" term_id="api-group" >}} are enabled. For details,
+see the `--feature-gates` and `--runtime-config` [kube-apiserver
```

Comment:
There are 2 points that I'd mention here:
1. I don't see any layout issues in the blogs mentioned on the Kubernetes website _(specifically for the [blog](https://kubernetes.io/blog/2022/12/15/dynamic-resource-allocation/) for which changes have been applied)_
2. The above changes are refactoring the existing code. However, the new changes don't reflect any change in the layout.